src/app/support/components/support-page/support-page.component.scss

```diff
@@ -0,0 +1,92 @@
+.support {
+  margin-top: 90px;
+
+  &-hero {
+    position: relative;
+    background-image: url(/assets/img/support/hero/support-hero__background.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    padding: 150px 0;
+    z-index: 0;
+
+    &::after {
+      content: "";
+      display: block;
+      top: 0;
+      bottom: 0;
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      background: rgba(53, 53, 53, 0.3);
+      z-index: -2;
+    }
+
```

Comment:
@k1k8r as far as I understood you try to follow BEM methodology in the project but the similar nested styles are not correspond with it. You combine SASS possibilities and BEM which is not best way.
Will be better to write classes and doesn't make a nested structure.